controlplane/kubeadm/api/v1beta1/kubeadmcontrolplanetemplate_types.go

```diff
@@ -28,6 +28,7 @@ type KubeadmControlPlaneTemplateSpec struct {
 // +kubebuilder:object:root=true
 // +kubebuilder:resource:path=kubeadmcontrolplanetemplates,scope=Namespaced,categories=cluster-api
 // +kubebuilder:storageversion
+// +kubebuilder:printcolumn:name="Cluster",type="string",JSONPath=".metadata.labels['machine\\.cluster\\.x-k8s\\.io/cluster-name']",description="Cluster"
```

Comment:
```suggestion
```
We don't set a cluster-name label on this resource

Reply:
Agreed, this is a template introduced for ClusterClasses, so it is shared across many clusters.